packages/x-date-pickers/src/DesktopNextDatePicker/DesktopNextDatePicker.test.tsx

```diff
@@ -55,11 +55,11 @@ describe('<DesktopNextDatePicker />', () => {
     expect(handleChange.callCount).to.equal(1);
   });
 
-  it('prop `showToolbar` – renders toolbar in desktop mode', () => {
```

Comment:
For the test files of the new pickers, can we follow the nomenclature of the other slots

```tsx
describe('Component slots: Popper', () => {
```

Can be adapter if needed, I just want to keep the consistency :+1: